crypto/ec/ec_key.c

```diff
@@ -492,6 +542,11 @@ int ossl_ec_key_public_check(const EC_KEY *eckey, BN_CTX *ctx)
         ERR_raise(ERR_LIB_EC, ERR_R_EC_LIB);
         goto err;
     }
+    /* IG 10.3.A additional comment 1: Perform a second check on the public key */
+    if (!EC_POINT_mul(eckey->group, point, NULL, eckey->pub_key, order, ctx)) {
```

Comment:
I am not seeing what this comment is referring to in the IG. please update if it is not correct,

Reply:
??

Reply:
Dunno here.  Omiting the IG reference also addresses your point :smirk:

Reply:
Please ask keypair - I couldnt find what this referred to..

Reply:
Wont hold it up on this - but it would be good to know where this came from.